### PR TITLE
subscription: Replace helpFile property with help_id

### PR DESCRIPTION
**Pull request by M4rtinK**
Previously we used to have one help content file per screen,
which no longer works for the new help content, which is using
a single docbook file with anchors.

So replace the helpFile property with a help_id property, which
holds a unique string id corresponding to one screen needing help.

The help id can then be mapped into an anchor in the main docbook file,
or even something else if needed.

Based on 56777977c9bfbd97b97ef0d72e59d0702f7b725c.

Related: rhbz#1593723
Related: rhbz#1691319

### CODE REVIEW

File: pyanaconda/ui/tui/spokes/__init__.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def input(self, args, key):
         # TRANSLATORS: 'h' to help
         if key.lower() == Prompt.HELP:
             if self.has_help:
-                help_path = get_help_path(self.helpFile, True)
+                help_path = get_placeholder_path(plain_text=True)
```
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/__init__.py:104]: Please, just call get_help_path with None for now.

File: pyanaconda/ui/tui/hubs/__init__.py [hunk 2]
```diff
@@ -139,7 +139,7 @@ def input(self, args, key):
             # TRANSLATORS: 'h' to help
             elif key == Prompt.HELP:
                 if self.has_help:
-                    help_path = get_help_path(self.helpFile, True)
+                    help_path = get_placeholder_path(plain_text=True)
```
Comment (poncovka) [inline pyanaconda/ui/tui/hubs/__init__.py:142]: Same here.

File: pyanaconda/ui/gui/__init__.py [hunk 3]
```diff
@@ -1028,7 +1029,7 @@ def _on_continue_clicked(self, window, user_data=None):
     def _on_help_clicked(self, window, obj):
         # the help button has been clicked, start the yelp viewer with
         # content for the current screen
-        start_yelp(get_help_path(obj.helpFile))
+        start_yelp(obj.help_id)
```
Comment (poncovka) [inline pyanaconda/ui/gui/__init__.py:1032]: Same here.

**Comment by M4rtinK** [comment]
In the end the help ids were handled in #2567.

**Pull request closed without merging**